docs/accessing_etcd.md

```diff
@@ -8,4 +8,7 @@ localkube host ip:  10.0.2.15
 You can use the host-ip:`10.0.2.15` to access localkube's resources, for example its etcd cluster.  In order to access etcd from within a pod, you can run the following command inside:
 ```shell
 curl -L -X PUT http://10.0.2.15:2379/v2/keys/message -d value="Hello"
-```
\ No newline at end of file
+```
+
+## Accessing Host Resources From Inside A Pod
+In order to access host resources from inside a pod, IP address `192.168.99.1` must be used.
```

Comment:
I think this actually depends on the driver you are using. Could you expand this to explain which driver you are using, and we can try to fill in the rest later?

Reply:
Thank you for your comment! Updated this @dlorenc